### ISSUE TITLE
Literal tabs or mixing tabs and spaces in 12.2 release

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5959]
|<img alt="schlenk's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| schlenk reported|
|-|-|
|Trac ID|trac#5959|
|Type|defect|
|Created|2012-09-11 11:22:16Z|
|Branch|https://github.com/twisted/twisted/tree/fewer-tabs-5959|

These files contain literal \x10 instead of \t, does trip some tab/spaces checkers, and mechanical replacing the tabs by spaces would even break the code.

 * twisted/names/test/test_hosts.py
 * twisted/scripts/tap2deb.py

This file mixes tabs and spaces.
 * twisted/trial/test/test_reporter.py

Found in 12.2. release tarball.

<details><summary>Searchable metadata</summary>

```
trac-id__5959 5959
type__defect defect
reporter__schlenk schlenk
priority__normal normal
milestone__ 
branch__branches_fewer_tabs_5959 branches/fewer-tabs-5959
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1347362536000000 1347362536000000
changetime__1347569273000000 1347569273000000
version__None None
owner__exarkun exarkun
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [35580]) Branching to 'fewer-tabs-5959'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [35581]) Remove some trailing whitespace, replace some indentation tabs with spaces, and replace some tabs in string data with tab escapes

leave the leading tabs in tap2deb alone because they're part of a makefile and tabs mean something in a makefile.  also there is no automated coverage for tap2deb.

refs [#5959](https://github.com/twisted/twisted/issues/5959)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

Someone else can worry about `tap2deb.py`, I don't want to make any more changes to it than I did.

[Build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/fewer-tabs-5959)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set owner to @exarkun|
|-|-|

I've added test coverage for `tab2deb` in a different branch that is up for review.

Please merge, thanks.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [35645]) Merge fewer-tabs-5959 

Author: exarkun
Reviewer: thijs
Fixes: [#5959](https://github.com/twisted/twisted/issues/5959)

Clean up some whitespace and replace some literal tabs with escape characters to avoid
ambiguity.
